### DISCUSSION TITLE
How can I specify the file storage location of the database

### DISCUSSION THREAD

**Discussion by TangStudio** [isar/isar#1365] [Help]
Where is the default file storage location of the database.I want to specify that the database file is stored in the specified file path of the phone, so that the database can still be read even after the APP is deleted and re-installed

**Comment by ilovelinux** [comment]
Change directory value in `Isar.open`.

Ref:
- Doc: https://isar.dev/tutorials/quickstart.html#_4-open-isar-instance
- Api: https://pub.dev/documentation/isar/latest/isar/Isar/open.html

**Reply by TangStudio** [reply]
thanks
